docs/src/SUMMARY.md

```diff
@@ -22,13 +22,17 @@
     - [Five minutes of theory](./apalache/theory.md)
 - [TLC Configuration Files](./apalache/tlc-config.md)
 - [Types and Annotations](./apalache/types-and-annotations.md)
-- [Snowcat: the New Type Checker](./apalache/typechecker-snowcat.md)
+- [The New Type Checker Snowcat](./apalache/typechecker-snowcat.md)
 - [Supported Features](./apalache/features.md)
 - [TLA+ Preprocessing](./apalache/preprocessing.md)
 - [Fine Tuning](./apalache/tuning.md)
 - [KerA: kernel logic of actions](./apalache/kera.md)
 - [Assignments in Apalache](./apalache/assignments.md)
 
+# HOWTOs
+
+- [How to write type annotations](./HOWTOs/howto-write-type-annotations.md)
```

Comment:
I wonder if this shouldn't just be part of the snowcat sections? Perhaps snowcat needs it's own directory and sub-chapters?

Lots of things int he manual describe how to do things: how to install, how to fine-tune, how to configure. I feel like howto sections often become weird grabbags, and it'd bet cleaner just to structure the manual by topic.

Reply:
Right. We have to restructure the docs. The reason why I called it a HOWTO is that I wanted to try this approach: https://documentation.divio.com

Reply:
Looking at our current manual, it mixes a tutorial, HOWTOs and explanations. That's why it looks quite weird.